WormsApp/board.cpp

```diff
@@ -48,7 +47,7 @@ void Board::update(int id, int oldX, int oldY, int newX, int newY) {
        return;
   int targetId = board_[newX][newY];
   if (targetId != 0 && targetId != id)
-  killed_.insert(targetId);
+   killed_.insert(targetId);
```

Comment:
You should keep indentation consistent - I used two-space indentation, you seem to be using four-space indentation and here you inserted only a single space.